assets/js/b24-form-interactions.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by iampamungkas on 8/19/17.
+ */
+'use strict';
+var onFormLoadInteractions = {
+     onLoadUnloadChange: function() {
+      var nodeId = document.getElementById("form_bit24");
```

Comment:
`document.getElementById` returns an object, not an integer, so `nodeId` is not an appropriate variable name.